apps-rendering/src/components/Layout/Layout.stories.tsx

```diff
@@ -206,6 +207,18 @@ export const DeadBlog = (): ReactElement => (
 );
 DeadBlog.story = { name: 'DeadBlog ' };
 
+export const GalleryItem = (): React.ReactNode => {
+	return (
+		<Gallery item={{ ...article, design: ArticleDesign.Gallery }}>
```

Comment:
Just FYI, I think galleries only ever include images in the body, whereas I believe this fixture has text, atoms, pullquotes etc. The snapshot may end up looking a bit unusual because we won't have introduced gallery styles for any of those other elements (because we don't need to)?

One solution to this might be to introduce a gallery fixture that only contains images. This could be done in a follow-up PR if necessary?

Reply:
Yeah I'm going to remove this for now, at one point I was thinking it might be easier to build and test in storybook and then discovered that the fixture isn't there and that would be a lot of work that I'll most likely need your help on.